torch/_dynamo/variables/builtin.py

```diff
@@ -1172,6 +1172,25 @@ def call_id(self, tx, *args):
         else:
             unimplemented(f"call_id with args {args}")
 
+    def _raw_value_to_format(self, value):
+        if value.is_python_constant():
+            return value.as_python_constant()
+        elif isinstance(value, variables.TensorVariable):
+            return value.get_real_value()
+        elif isinstance(value, variables.BaseListVariable):
```

Comment:
I think this has been covered by ```value.is_python_constant()```.

Reply:
I don't think so, `is_python_constant` will fail for a list of tensors, I think.

Reply:
```is_python_constant``` won't fail, it returns ```False```.
https://github.com/pytorch/pytorch/blob/efc3887ea508b3cfd94603fd8afe4e8cf6dce7b7/torch/_dynamo/variables/base.py#L152-L157